File: design-patterns/go/abstract_factory/transport/bicycle.go
// bicycle.go

package main

import "fmt"

type Bicycle struct {
}

func NewBicycle() (bicycle *Bicycle) {
	bicycle = &Bicycle{}
	return
}

func (bicycle *Bicycle) Start() {
	fmt.Println("Bicycle Started")
}

func (bicycle *Bicycle) Stop() {
	fmt.Println("Bicycle Stopped")
}

func (bicycle *Bicycle) Repair() {
	fmt.Println("Bicycle Repair")
}

File: design-patterns/php/abstract-factory/transport/Bicycle.php
<?php
// Bicycle.php

namespace AbstractFactory\Transport;


class Bicycle implements Transport {
    public function start(): void {
        echo "Bicycle Started\n";
    }

    public function stop(): void {
        echo "Bicycle Stopped\n";
    }

    public function repair(): void {
        echo "Bicycle Repair\n";
    }
}